src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
so unless we think there will be an orientation and/or position even when hasXXX is false (if they even continue to exist) why would we not stick with looking at the actual variable we intend to use?  it seems clear to me in these last few comments that there is a loose convention but not actual specification, so I think it makes sense to be slightly defensive... at least until there is an actual spec agreed and implemented by browsers we intent to support (see above proposal, and I would also add Edge if actual HMDs were available)